bot/exts/fun/madlibs.py

```diff
@@ -63,7 +64,11 @@ async def madlibs(self, ctx: commands.Context) -> None:
         def author_check(message: discord.Message) -> bool:
             return message.channel.id == ctx.channel.id and message.author.id == ctx.author.id
 
-        original_message = await ctx.send("Loading your Madlibs game...")
+        loading_embed = discord.Embed(
+            title="Madlibs", description="Loading your Madlibs game...", color=Colours.python_blue
+        )
+        original_message = await ctx.send(embed=loading_embed)
+        time.sleep(1)
```

Comment:
This will block the rest of the bot, besides, why sleep?

Reply:
Why will this block the bot?

Reply:
I thought `sleep` would make it more realistic that it's "loading the game", in a sense.

Reply:
> I thought `sleep` would make it more realistic that it's "loading the game", in a sense.

No we don't want it to be realistic, the faster the better. It is i just for getting a message to start with.

Reply:
Alright, I will remove it then.